spec/controllers/api/v1/users_controller_spec.rb

```diff
@@ -186,25 +186,6 @@
       end
 
       describe "search" do
-        context "by display_name" do
```

Comment:
it wasn't failing since the display name matches the login trigram threshold based on factory data. I left it in to show that searching with different attribute inputs can still match the login trigram data based on how similar the login / display_name data actually is, e.g camallen / cam allen